ambry-frontend/src/main/java/com/github/ambry/frontend/FrontendRestRequestService.java

```diff
@@ -213,11 +213,11 @@ public void start() throws InstantiationException {
     namedBlobsCleanupRunner = new NamedBlobsCleanupRunner(router, namedBlobDb);
     if (frontendConfig.enableNamedBlobCleanupTask) {
       namedBlobsCleanupScheduler = Utils.newScheduler(1, "named-blobs-cleanup-", false);
-      int oneWeekInSeconds = 60 * 60 * 24 * 7;
-      int initialDelayInSeconds = random.nextInt(oneWeekInSeconds);
+      int oneDayInSeconds = 60 * 60 * 24;
```

Comment:
run it every 7 days would be good, or we can put it in a configuration.

Reply:
I updated the schedule interval to be config-driven with weekly schedule.